From 590928882d3c33e559b6f523b6d6f0fcf353ba9d Mon Sep 17 00:00:00 2001
From: Byron McCollum <byron.mccollum@rackspace.com>
Date: Sat, 16 Jan 2016 22:09:50 -0600
Subject: [PATCH] Adjust permission of /etc/ceph to allow Cinder to create Ceph
 Volumes

Cinder Volume needs access to /etc/ceph to create Ceph Volumes. This
change passes `conf_directory_group` and `conf_directory_mode` as
role parameters to adjust permissions accordingly.

Note: This depends on an upstream pull request for the ceph-common role

https://github.com/ceph/ceph-ansible/pull/485

Fixes: #747
---
 rpcd/playbooks/ceph-osd.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpcd/playbooks/ceph-osd.yml b/rpcd/playbooks/ceph-osd.yml
index 23f7e36..d86ca3d 100644
--- a/rpcd/playbooks/ceph-osd.yml
+++ b/rpcd/playbooks/ceph-osd.yml
@@ -43,5 +43,6 @@
         - ceph-logs
   roles:
     - ceph-osd
+    - { role: "ceph-common", conf_directory_group: "cinder", conf_directory_mode: "654" }
   vars:
     is_metal: "{{ properties.is_metal|default(false) }}"
-- 
2.7.0

